install.sh

```diff
@@ -565,19 +565,19 @@ systemd_disable() {
 
 # Compose firewall rule
 firewall_rule() {
```

Comment:
I prefer this:
```sh
# Compose firewall rule
firewall_rule() {
  _firewall_path=$(command -v "$FIREWALL" 2>&1 || :)

  case $FIREWALL in
    firewall-cmd)
      printf "%s\n%s\n" \
        "$_firewall_path --add-port=$NODE_EXPORTER_PORT/tcp --permanent" \
        "$_firewall_path --reload"
      ;;
    ufw)
      printf "%s\n" \
        "$_firewall_path allow $NODE_EXPORTER_PORT/tcp"
      ;;
    iptables)
      printf "%s\n" \
        "$_firewall_path -A INPUT -p tcp --dport $NODE_EXPORTER_PORT -m state --state NEW -j ACCEPT"
      ;;
    *) fatal "Unknown firewall '$FIREWALL'" ;;
  esac
}
```

Note: `firewall_rule` is called only if a firewall exists. `|| :` and `*) fatal ...` are only for correctness.